image-service/.nvmrc.in

```diff
@@ -0,0 +1 @@
+${NODEJS_VERSION}
```

Comment:
Good catch Nuoye, this file is not used for now, it is copied from all other folders and reserved for future use. Yet I'm thinking whether it is a good idea to leave it as it is, and take a thorough look later- if it is no longer needed, I can open another PR and remove the files under all folders.